### PR TITLE
single trim and toList

### PR DESCRIPTION
**Pull request by christophsturm**

### CODE REVIEW

File: strikt-core/src/main/kotlin/strikt/assertions/CharSequence.kt [hunk 1]
```diff
@@ -225,3 +225,11 @@ fun <T : CharSequence> Builder<T>.isNotBlank(): Builder<T> =
  */
 val <T : CharSequence> Builder<T>.length: Builder<Int>
   get() = get(CharSequence::length)
+
+/**
+ * trims a CharSequence
+ *
+ * @see CharSequence.trim
+ */
+fun <T : CharSequence> Builder<T>.trim(): Builder<CharSequence> =
```
Comment (robfletcher) [inline strikt-core/src/main/kotlin/strikt/assertions/CharSequence.kt]: This should return `Builder<T>`

File: strikt-core/src/main/kotlin/strikt/assertions/Arrays.kt [hunk 2]
```diff
@@ -82,3 +82,10 @@ fun Assertion.Builder<CharArray>.contentEquals(other: CharArray): Assertion.Buil
   assertThat("array content equals %s", other) {
     it.contentEquals(other)
   }
+
+/**
+ * Maps an array to a list to make it possible to use the iterable matchers
+ * [CharArray.contentEquals].
```
Comment (robfletcher) [inline strikt-core/src/main/kotlin/strikt/assertions/Arrays.kt]: I don’t get what this link refers to.

File: strikt-core/src/main/kotlin/strikt/assertions/CharSequence.kt (hunk 1 above)
Comment (christophsturm) [reply strikt-core/src/main/kotlin/strikt/assertions/CharSequence.kt]: `CharSequence.trim` returns `CharSequence`, but i added a version for string to.

File: strikt-core/src/main/kotlin/strikt/assertions/Arrays.kt (hunk 2 above)
Comment (christophsturm) [reply strikt-core/src/main/kotlin/strikt/assertions/Arrays.kt]: that was just a copy paste fail.

File: strikt-core/src/main/kotlin/strikt/assertions/CharSequence.kt (hunk 1 above)
Comment (robfletcher) [reply strikt-core/src/main/kotlin/strikt/assertions/CharSequence.kt]: ah. Thanks.

**Pull request merged**